rosbag2_py/README.md

```diff
@@ -14,7 +14,12 @@ source install/setup.sh
 # Make sure rosbag2_py can be imported
 python3 -c 'import rosbag2_py'
 
-sudo apt update && sudo apt install mypy=0.942-1ubuntu1
+# Ubuntu 24.04
+sudo apt update && sudo apt install mypy
+
+# Older Ubuntu
+# sudo pip3 install -U --break-system-packages mypy==1.9
```

Comment:
@r7vme The problem is that on Ubuntu 22.04 the default pip3 version is 22.0.2 and python version is 3.10 and there is no `--break-system-packages` option for pip3 install on those versions.

Reply:
@MichaelOrlov 

> The problem is that on Ubuntu 22.04 the default pip3 version is 22.0.2 and python version is 3.10 and there is no --break-system-packages option for pip3 install on those versions.

My bad, `--break-system-packages` should be removed

> What version of the mypy will be used? It seems like an undefined behavior that we would like to avoid.

I tested in docker container with ubuntu:jammy and after installing 1.9.0 via pip and 0.942 via apt. I got

```
root@f647688bc269:/# mypy --version
mypy 1.9.0 (compiled: yes)
```

Reply:
@r7vme Ok it works. 
Only one concern is about `sudo` for `pip3 install` command.
It seems we can install mypy without `sudo`.

Reply:
Without sudo mypy installing in the user local folder. e.g. `/home/morlov/.local/bin/mypy`